pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
     "google-auth==2.19.1",
     "casbin_redis_adapter==1.2.0",
     "casbin_pymongo_adapter==1.1.0",
-    "sat-utils==1.5.0",
+    "sat-utils>=1.5.0, <2.0.0",
```

Comment:
is there a reason you're opening up a range here rather than locking to the most recent version of sat-utils? 1.7.0 I believe